src/FSharpVSPowerTools.Logic/Navigation/GoToDefinitionFilter.fs

```diff
@@ -33,6 +28,12 @@ type internal UrlChangeEventArgs(url: string) =
     inherit EventArgs()
     member __.Url = url
 
+[<NoComparisonAttribute; NoEquality>]
+type GoToDefinitionResolution =
```

Comment:
I was considering the NotFound case but in current implementation (in asyncMaybe) it still makes sense to return Some/None, I'll give it another look.

Also `SnapshotSpan` will be going away from `External` case as I'll be working on making the logic work without Visual Studio APIs.